core/blockchain.go

```diff
@@ -1768,6 +1768,10 @@ func (bc *BlockChain) insertChain(chain types.Blocks, setHead bool) (int, error)
 			if !bc.stateCache.InTransition() && !bc.stateCache.Transitioned() {
 				bc.stateCache.StartVerkleTransition(parent.Root, emptyVerkleRoot, bc.Config(), bc.Config().PragueTime, parent.Root)
 			}
+		} else {
+			// If the verkle activation time hasn't started, declare it as "not started".
+			// This is so that
+			bc.stateCache.InitTransitionStatus(false, false)
```

Comment:
Should the correct place to change this be here? https://github.com/gballet/go-ethereum/blob/transition-post-genesis/core/blockchain.go#L329

Reply:
well for one thing the comment above is outdated.

Reply:
TLDR no it doesn't. The reason for this is that this activation comes from the miner, AFTER the blockchain has been initialized. It's also true of a reorg: if for whatever reason the verkle fork time changes (let's say it's pushed back) the reorg will have to go through without the chain to be reinitialized, and so `NewBlockchain` will not be called.